src/main/resources/default/extensions/wondergem-page/storage-vfs-modals.html.pasta

```diff
@@ -2,196 +2,214 @@
-                                    '<td class="align-right">' + child.lastModifiedString + '</td>' +
-                                    '</tr>');
-
-                                if (child.directory) {
-                                    $tr.find('.file-link i.fa').addClass('fa-folder-open');
-                                } else {
-                                    $tr.find('.file-link i.fa').addClass('fa-file');
-                                }
+                            _modal.querySelector(".search-result-js").style.display = _searchForm.value ? "block" : "none";
 
-                                $tr.find('.file-link').click(function () {
-                                    if ($(this).data('dir')) {
-                                        update($(this).data('path'));
-                                    } else {
-                                        $modal.modal('hide');
-                                        resolve($(this).data('path'));
-                                    }
-                                });
-                                $tr.appendTo($table);
+                            if (json.children.length <= pageSize) {
+                                pagination._nextBtn.classList.add("disabled");
+                            } else {
+                                pagination._nextBtn.classList.remove("disabled");
                             }
 
-                            var $breadcrumbs = $modal.find('.breadcrumb');
-                            $breadcrumbs.html('');
-                            for (var i = 0; i < json.path.length; i++) {
-                                var element = json.path[i];
-                                var name = element.name === '/' ? '<i class="fa fa-home"></i>' : element.name;
-
-                                if (restrictToPath !== undefined && !element.path.startsWith(restrictToPath)) {
+                            const _breadcrumbs = _modal.querySelector('.breadcrumb-js');
+                            _breadcrumbs.textContent = '';
+                            for (let i = 0; i < json.path.length; i++) {
+                                const element = json.path[i];
+                                if (!config.filter(element)) {
                                     continue;
                                 }
-
-                                var $li = $('<li><a class="file-link" href="#" data-path="' + element.path + '">' + name + '</a></li>');
-                                $li.find('.file-link').click(function () {
-                                    update($(this).data('path'));
+                                const _label = element.name === '/' ? '<i class="fa fa-home"></i>' : element.name;
+                                const _li = document.createElement("li");
+                                _li.innerHTML = '<a class="file-link" href="#" data-path="' + element.path + '">' + _label + '</a>';
```

Comment:
Shouldn't we use Mustache to generate inner html? this is a simple case, but we also have more complex ones....